patches/third_party-blink-renderer-modules-plugins-dom_plugin_array.cc.patch

```diff
@@ -1,16 +1,24 @@
 diff --git a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
-index 2bb7e4138da4e9b31734e4ee3b5e26e7399bcffd..98d7ac259179dab1360cb6b6dda26b2bc287d4b8 100644
+index 2bb7e4138da4e9b31734e4ee3b5e26e7399bcffd..ae62755ab4fecab3b037081b17ccbb12a0f128c7 100644
 --- a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
 +++ b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
-@@ -196,6 +196,7 @@ void DOMPluginArray::UpdatePluginData() {
+@@ -170,6 +170,7 @@ HeapVector<Member<DOMMimeType>> DOMPluginArray::GetFixedMimeTypeArray() {
+ }
+ 
+ bool DOMPluginArray::IsPdfViewerAvailable() {
++  BRAVE_DOM_PLUGINS_IS_PDF_VIEWER_AVAILABLE
+   DCHECK(should_return_fixed_plugin_data_);
```

Comment:
let's just delete this `DCHECK`. It become obsolete after [this change](https://github.com/chromium/chromium/commit/c2abba012946a341b479821594eccaf45c893b3f), Chromium should fix it at some point.

Reply:
done in https://github.com/brave/brave-core/pull/12989/commits/e3c3f42e8bdfa12aea0acda5546fe6fe645ec585